CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Changelog
 
+## 0.1.19
```

Comment:
Should we aligned it with the internal version?

Reply:
@gregwym I wasn't sure if we should skip so many versions? I saw this was a few off from the internal version but wasn't sure if it was right to skip 3 versions from 0.1.8 -> 0.1.21

cc @jordanwnolan wdyt?